### PR TITLE
update setCertificateVerifyProc example

### PR DESCRIPTION
**Pull request by herrmannplatz**
setCertificateVerifyProc example should reflect new api

### CODE REVIEW

**Comment by herrmannplatz** [comment]
Not sure whether shorthand if should be used or not.

File: docs/api/session.md [hunk 1]
```diff
@@ -274,8 +274,13 @@ verify proc.
 const {BrowserWindow} = require('electron')
 let win = new BrowserWindow()
 
-win.webContents.session.setCertificateVerifyProc((hostname, cert, callback) => {
-  callback(hostname === 'github.com')
+win.webContents.session.setCertificateVerifyProc((request, callback) => {
+  const {hostname} = request
+  if(hostname === 'github.com') {
```
Comment (kevinsawicki) [inline docs/api/session.md]: Space needed after `if`, it is currently failing on CI due to linting:

```
  api/session.md
         279:3:    Expected space(s) after "if".
```

**Comment by kevinsawicki** [comment]
Thanks @herrmannplatz 👍 🚀

**Comment by herrmannplatz** [comment]
merci

**Pull request merged**